### PR TITLE
HW1: Part 1 & 2

### PR DESCRIPTION
**Pull request by ulasonat**
PR for updating the ua2182.md

### CODE REVIEW

**Comment by ulasonat** [comment]
Note that I had the PR from HW0 still open by the time of this and I closed that one to create this PR.

File: tkp2108.md [hunk 1]
```diff
@@ -4,11 +4,13 @@ Name: Tim Paine
 
 UNI: tkp2108
 
-Github: [link](https://github.com/timkpaine)
+GitHub: [link](https://github.com/timkpaine)
```
Comment (timkpaine) [inline tkp2108.md:7]: undo these changes

**Review by timkpaine — CHANGES_REQUESTED** [review]
Readme looks good, just need to make markdown files for your project and 1 of the projects to which you want to contribute

**Comment by ulasonat** [comment]
I had two separate pull requests for HW1, which now I see was not necessary. I close this down, feel free to ignore this. My other pull request is named HW1, which I worked on branch hw1_p3!

**Pull request closed without merging**